app/assets/javascripts/controllers/container_dashboard/pf-fixes.js

```diff
@@ -0,0 +1,7 @@
+angular.module('patternfly.card').run(['$templateCache', function($templateCache) {
+  'use strict';
+
+  /* Fixes aggregate-status-card in patterfly 2.4.0 - Remove with next version.*/
```

Comment:
@zeari please see #5180 (update pattenfly & angular-patternfly to 2.5.0), this is probably no longer necessary..